ui/imports/shared/popups/walletconnect/PairWCModal/WCUriInput.qml

```diff
@@ -76,12 +78,18 @@ ColumnLayout {
 
             readonly property bool showIcon: input.valid && input.text.length > 0
 
+            StatusLoadingIndicator {
+                anchors.centerIn: parent
+                color: Theme.palette.green
```

Comment:
Probably not `green`

<img width="604" alt="Screenshot 2024-06-26 at 14 03 16" src="https://github.com/status-im/status-desktop/assets/47811206/d937d3c6-dfda-4e6a-aa4a-680aa6730e3e">